xml/System.Data.Spatial/DbGeography.xml

```diff
@@ -773,8 +773,8 @@
         <ReturnType>System.Nullable&lt;System.Double&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the Latitude coordinate of this DbGeography value, if it represents a point. &lt;returns&gt;The Latitude coordinate value of this geography value, if it represents a point; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <value>The Latitude coordinate of this DbGeography value.</value>
+        <summary>Gets the Latitude coordinate of this DbGeography value, if it represents a point.</summary>
+        <value>The Latitude coordinate of this DbGeography value, if it represents a point; otherwise <see langword="null" />.</value>
```

Comment:
```suggestion
        <value>The Latitude coordinate of this DbGeography value, if it represents a point; otherwise, <see langword="null" />.</value>
```